src/trace-api.js

```diff
@@ -165,11 +166,17 @@ TraceAgent.prototype.runInRootSpan = function(options, fn) {
     // Create a new root span, and invoke fn with it.
     var traceId = incomingTraceContext.traceId || (uuid.v4().split('-').join(''));
     var parentId = incomingTraceContext.spanId || '0';
+    var frameOffset;
+    if (semver.satisfies(process.version, '>=8')) {
```

Comment:
Thanks for the suggestion! semver checks on the critical path should now be gone.